webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -39,21 +40,30 @@ class ResponsePage extends Component {
             applicationFormService.getForEvent(this.props.event.id),
             responsesService.getResponseDetail(this.props.match.params.id, this.props.event.id),
             tagsService.getTagList(this.props.event.id),
-            reviewService.getReviewAssignments(this.props.event.id)
         ]).then(responses => {
             console.log(responses);
             this.setState({
                 eventLanguages: responses[0].event ? Object.keys(responses[0].event.name) : null,
                 applicationForm: responses[1].formSpec,
                 applicationData: responses[2].detail,
                 tagList: responses[3].tags,
-                reviewers: responses[4].reviewers,
-                error: responses[0].error || responses[1].error || responses[2].error || responses[3].error || responses[4].error,
-                isLoading: false
-            }, this.handleData);
-        });
+                reviewers: responses[2].detail.reviewers,
+                error: responses[0].error || responses[1].error || responses[2].error || responses[3].error,
+            }, this.handleData, this.getReviewResponses(responses[2].detail, responses[2].detail.reviewers));
+    });
     };
 
+    getReviewResponses(applicationData, reviewers) {
+        reviewers.map((val, index) => {
+            let num = index + 1;
+            reviewService.getResponseReviewAdmin(applicationData.id, this.props.event.id, val.reviewer_user_id)
```

Comment:
Since you've made a specific backend API for this feature, it probably makes sense to return all the reviews in one call instead of having multiple calls for each review. This would be more efficient. That way you can also wrap it up in an object that maybe has a single review_form (with all the questions) and a list of all the review responses.